examples/platform-specific/zoul/orion/client/Makefile.target

```diff
@@ -1 +1,2 @@
 TARGET = zoul
+BOARD = orion
```

Comment:
@simonduq made a comment about this one not being orion-specific (if that's the case then I do find the path misleading). I am not sure myself, I did not look into it.

Reply:
Yes it looks like `ip64-router` is meant for the Orion, but `client` is just an example HTTP client used to try out an IP64 Orion router.

Reply:
Let's then just undo my change to the Makefile.target and merge.

Reply:
Removed.